lib/Authentication.php

```diff
@@ -102,10 +106,13 @@ public function requestLWAToken(): array
             }
             $jsonData["refresh_token"] = $this->lwaRefreshToken;
         }
-
-        $res = $this->client->post($this->lwaAuthUrl, [
-            \GuzzleHttp\RequestOptions::JSON => $jsonData,
-        ]);
+        
+        $lwaTokenRequestHeaders = [
+            'Content-Type' => 'application/json',
+        ];
+        $lwaTokenRequestBody = \GuzzleHttp\json_encode($jsonData);
+        $lwaTokenRequest = new Psr7\Request('POST', $this->lwaAuthUrl, $lwaTokenRequestHeaders, $lwaTokenRequestBody);
+        $res = $this->client->send($lwaTokenRequest);
```

Comment:
What's the purpose of these changes?

Reply:
to fit `\GuzzleHttp\ClientInterface` it has no `post` method that was used before,
also it gives more flexibility as benefit -> not only `GuzzleHttp\Client` object can be used now it can be anything that implements `GuzzleHttp\ClientInterface` interface

Reply:
You are welcome!